doc/development.md

```diff
@@ -35,24 +35,46 @@ Create virtual environment for vdsm:
 
 ## Building Vdsm
 
-To configure sources (run `./configure --help` to see configuration options):
+To configure sources (create Makefiles and configure script): 
 
     git clean -xfd
     ./autogen.sh --system --enable-timestamp
-    make
 
-To test Vdsm (refer to tests/README for further tests information):
+(It is recommended to start fresh after an update, from the previous step, before
+building RPMs, because the generated Makefile also contains the release number.)
 
-    make check
+Now you can create the virtual environment (https://docs.python.org/3/library/venv.html),
+which is necessary to run the tests later. This needs to be done only once:
+
+    make venv
+
+To see configuration options run `./configure -h`. Configure things, if necessary,
+then start make to build Vdsm:
```

Comment:
Changing the configuration is advanced topic that always never needed. Lets keep things
simpler here, and keep this info in another section.

Reply:
You mean we need not mention that there are options here? Or we don't talk about configure at all since it's optional?

Reply:
There is no need to talk about configure in this section. Maybe in another "advanced" section at the end of the document.